src/jep/InvocationHandler.java

```diff
@@ -40,6 +40,7 @@
     private long target;
 
     private Jep jep;
+    private final boolean isDirect;
```

Comment:
I am a little lost as to what isDirect is referring to.  My head is stuck on direct memory.  Please explain.